src/containers/Stablecoins/utils.ts

```
@@ -113,23 +113,27 @@ export const buildStablecoinChartData = ({
 		}
 	}
 
-	const peggedAreaChartData = Object.entries(unformattedAreaData).map(([date, chart]) => {
-		if (typeof chart === 'object') {
+	const peggedAreaChartData = Object.entries(unformattedAreaData)
+		.sort(([a], [b]) => Number(a) - Number(b))
+		.map(([date, chart]) => {
```

Biome (2.3.13)

[error] This callback passed to map() iterable method should always return a value.

Add missing return statements so that this callback returns a value on all execution paths.

(lint/suspicious/useIterableCallbackReturn)